packages/blade/src/components/Box/BaseBox/baseBoxStyles.ts

```diff
@@ -175,7 +175,10 @@ const getAllProps = (
     props.borderBottomWidth ||
     props.borderTopWidth ||
     props.borderLeftWidth ||
-    props.borderRightWidth;
+    props.borderRightWidth ||
+    props.borderWidth ||
```

Comment:
`borderStyle` was not being added when we pass just `borderWidth` or `borderColor`